Source/SvgElement.cs

```diff
@@ -512,6 +512,17 @@ protected virtual void WriteStartElement(XmlWriter writer)
                 else
                 {
                     var prefix = writer.LookupPrefix(this.ElementNamespace);
+                    if (prefix == null && !this.ElementNamespace.Equals(SvgNamespaces.SvgNamespace))

```

Comment:
Is there a reason it's `Equals` here and `==` on line 519 ?

Reply:
I agree, one way to do it for both comparisons would be more clean. I've updated it in my latest commit. Good catch.